### PR TITLE
Project Capstone2 JavaScript

### PR DESCRIPTION
**Pull request by DeVuDeveloper**
Hi, 
All project's requirements done.
- Code partner review dayly or more time daily.
- Kanban cards all done.
 Vujovic Dejan: student A
- Item rendered dinamically, retrieved from API
- function for counter made and passed jest tests
- functionality for likes done. If user click on hear button, data sent to Involvement API and back, right numbers of likes 
- displayed for every items
Aniekan Udo: student B
-when user click on comments button pop-up appears with detasiled info of item, collected from API.
- made functionality for comments
- when user send comment, data is sent to Involvement API and back to user. All comments are displayed for right item.
- function counter for comments made and passed all tests
Thank You.

Update: Updated readme with
- cd-path to project
- access for video set to anyone

Jest test all checks passed!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,74 @@
...
+* webpack
+* jest
+
+	
+## Setup
+To run this project, install it locally using:
+- cd Desktop
+- git clone https://github.com/VuDej/Capstone-2-JavaScript.git
+- npm install
+- npm start
```
Comment (Deepakdanger) [inline README.md:37]: - [ ] I think you missed one step here which is important. After `git clone ....`, you have to go to the correct folder by entering the command `cd Capstone-2-JavaScript`. Please include this.

File: README.md [hunk 2]
```diff
@@ -1 +1,74 @@
...
\ No newline at end of file
+# Capstone 2 Javascript
+## Fish recipes
+
+## Live preview [here](https://vudej.github.io/Capstone-2-JavaScript/dist/). 
+
+##  Video
+
+Quick review of page and code.
+[video link](https://drive.google.com/drive/u/0/folders/1C0FNMW0G4Pz6nbGOo2LsWOHNexgSbzxa)
```
Comment (Deepakdanger) [inline README.md:9]: - [ ] Currently, I won't be able to access the video link. Please include the link that anyone can see.

File: src/modules/test.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+  test('if it is right title', () => {
+    const receivedData = mockJsonFromAPI;
+    expect(receivedData[12].strMeal).toBe('Mediterranean Pasta Salad');
+  });
+
+  test('if it is right id', () => {
+    const receivedData = mockJsonFromAPI;
+    expect(receivedData[25].idMeal).toBe('52975');
+  });
```
Comment (Deepakdanger) [inline src/modules/test.js:19]: - [ ] One test are failing. Here we have to expect id to be integer and not string. Please fix this.

Screenshot from 2022-02-05 20-01-39

Hint:
`expect(receivedData[25].idMeal).toBe(52975);`

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @VuDej & @Anny85-code, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by EricMbouwe — APPROVED** [review]
### Hi @team, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**